### PR TITLE
feat: improvements on glossary Apple Safari

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

- it's now supported on visionOS
- more detailed information
- linked with glossary WebKit

### CODE REVIEW

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks, @PassionPenguin. I've added a few suggestions.

File: files/en-us/glossary/apple_safari/index.md [hunk 1]
```diff
@@ -6,12 +6,14 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-[**Safari**](https://www.apple.com/safari/) is a {{Glossary("Browser","Web browser")}} developed by Apple and bundled with macOS, iPadOS, and iOS. It's based on the open-source [WebKit](https://webkit.org/) engine.
+[**Safari**](https://www.apple.com/safari/) is a {{Glossary("Browser","web browser")}} developed by Apple and is built into Apple's operating systems, including macOS (for Mac computers), iPadOS (for iPad tablets), iOS (for iPhones), and visionOS (for augmented reality devices like the Apple Vision Pro).
+
+Safari was first introduced in 2003 on Mac OS X Panther and later on iOS in 2007, using the open source {{glossary("WebKit")}} engine, which was derived from [KHTML](https://en.wikipedia.org/wiki/KHTML). Apple used to offer a version of Safari for Windows, but it was discontinued in 2012.
```
Comment (Josh-Cena) [inline files/en-us/glossary/apple_safari/index.md:11]: This information is unnecessary for defining Safari and doesn't help readers understand what Safari is.

Comment (PassionPenguin) [reply files/en-us/glossary/apple_safari/index.md:11]: shouldn't rendering engine been related to what a browser is? Given readers on site care about the implementation of different features supported by engines themselves

Comment (Josh-Cena) [reply files/en-us/glossary/apple_safari/index.md:11]: Sure, but the history is irrelevant for a glossary entry and is more suitable for a Wikipedia-style article. The first paragraph is good enough.

Comment (dipikabh) [reply files/en-us/glossary/apple_safari/index.md:11]: Agree, let's remove this info

Comment (dipikabh) [inline files/en-us/glossary/apple_safari/index.md:11]:
```suggestion

```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Review by dipikabh — APPROVED** [review]

File: files/en-us/glossary/apple_safari/index.md [hunk 2]
```diff
@@ -6,12 +6,13 @@ page-type: glossary-definition
 
 {{GlossarySidebar}}
 
-[**Safari**](https://www.apple.com/safari/) is a {{Glossary("Browser","Web browser")}} developed by Apple and bundled with macOS, iPadOS, and iOS. It's based on the open-source [WebKit](https://webkit.org/) engine.
+[**Safari**](https://www.apple.com/safari/) is a {{Glossary("Browser","web browser")}} developed by Apple and is built into Apple's operating systems, including macOS (for Mac computers), iPadOS (for iPad tablets), iOS (for iPhones), and visionOS (for augmented reality devices like the Apple Vision Pro).
+
```
Comment (dipikabh) [inline files/en-us/glossary/apple_safari/index.md:10]:
```suggestion
```

File: files/en-us/glossary/apple_safari/index.md (hunk 1 above)
Comment (PassionPenguin) [reply files/en-us/glossary/apple_safari/index.md]: well, seems to have removed far more information than history itself. now the page contains nothing but OS it supports.

**Pull request merged**